README.md

```diff
@@ -1,3 +1,12 @@
+Running Certbot in Docker 
+=========================
+
+Docker is an amazingly simple and quick way to obtain a certificate. However, this mode of operation is unable to install certificates or configure your webserver, because our installer plugins cannot reach your webserver from inside the Docker container.
+ 
+**Most users should use the operating system packages (see instructions at certbot.eff.org) or, as a fallback, certbot-auto. You should only use Docker if you are sure you know what you are doing and have a good reason to do so.**
```

Comment:
This proposed change makes what the installation instructions are at https://certbot.eff.org a little more generic. I think this is useful because for some users their best option is to use certbot-auto and we're actively working on new methods of distributing Certbot and not including the details here makes it less likely for the text here to become incorrect or out-of-date.


```suggestion
**Most users should install Certbot by following the installation instructions at https://certbot.eff.org/instructions. You should only use Docker if you are sure you know what you are doing and have a good reason to do so.**
```